src/sass/_storage.scss

```diff
@@ -11,6 +11,10 @@
     width: 14rem;
   }
 
+  .size.clustered {
+    width: 20rem;
+  }
+
```

Comment:
This distribution of the column widths should be optimized. In the example below, the driver and two volumes columns can give space to allow showing the full content of the member labels.

![image](https://github.com/user-attachments/assets/e284389d-5350-43ea-9f89-b15a65ec11a2)

Reply:
This applies also to the detail page. We should allow the member names to grow wider in order not to truncate if the space is available.

![image](https://github.com/user-attachments/assets/6b5367b8-d52a-449b-86c6-4336039bb6a8)

Reply:
We might want to do some calculation based on the longest member name to find the best max width for them.